lib/driver.rb

```diff
@@ -0,0 +1,98 @@
+require 'csv'
+require_relative 'csv_record'
+
+module RideShare
+  class Driver < CsvRecord
+
+    attr_reader :name, :vin
+    attr_accessor :status, :trips
+    
+    def initialize(
+                  id:,
+                  name:,
+                  vin:,
+                  status: :AVAILABLE,
+                  trips: nil
+        )
+      super(id)
+
+      unless super(id) > 0 
+        raise ArgumentError
+      end
+
+      @name = name
+      @vin = vin
+
+      unless vin.length == 17
+        raise ArgumentError
```

Comment:
Including a message that contains the bad argument is helpful for debugging and your users.

```suggestion
        raise ArgumentError, "Bad vin: #{vin}"
```